content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Linux पर bash ऑटो-कम्पलीशन"
+description: "Linux पर bash ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक विन्विन्यास।"
+headless: true
+---
+
+## परिचय
+
+Bash के लिए kubectl  कम्प्लिशन स्क्रिप्ट `kubectl completion bash` कमांड के साथ उत्पन्न की जा सकती है। आपके शेल में कम्प्लिशन स्क्रिप्ट को सोर्स करने से kubectl ऑटोकम्प्लिशन ऐनेबल हो जाती है।
```

Comment:
We'd want to use a fixed spelling for `completion` : here we have used `कम्पलीशन` and `कम्प्लिशन` both. I suggest going forward with `कम्पलीशन`